libs/qCC_glWindow/ccRenderingTools.cpp

```diff
@@ -59,8 +60,8 @@ void ccRenderingTools::ShowDepthBuffer(ccGBLSensor* sensor, QWidget* parent/*=0*
 			}
 			else if (*_zBuff > 0)
 			{
-				maxDist = std::max(maxDist, *_zBuff);
-				minDist = std::min(minDist, *_zBuff);
+				maxDist = std::max(maxDist, static_cast<const ScalarType>(*_zBuff));
```

Comment:
I believe in this method, there was another mistake. minDist, maxDist and coef should all be of 'PointCoordinateType' type.